README.md

```diff
@@ -1,9 +1,11 @@
 # LaunchDarkly Server-Side SDK for Node.js
 
-# Important note
+# Use [@launchdarkly/node-server-sdk](https://www.npmjs.com/package/@launchdarkly/node-server-sdk) instead of this package.
```

Comment:
```suggestion
> [!IMPORTANT]
> Use [@launchdarkly/node-server-sdk](https://www.npmjs.com/package/@launchdarkly/node-server-sdk) instead of this package.
```

Reply:
![image](https://github.com/launchdarkly/node-server-sdk/assets/4955475/30975bca-4830-4d2b-ae9a-01cc4e427f2c)

I think, in this case, I am going to leave it out because the NPM side doesn't render it as well and NPM is the target here.